models/auth/oauth2.go

```diff
@@ -81,6 +82,8 @@ func Init(ctx context.Context) error {
 		builtinAllClientIDs = append(builtinAllClientIDs, clientID)
 	}
 
+	gob.Register(LinkAccountUser{})
```

Comment:
Why is this necessary?

Reply:
This is needed in order to encode and store the struct in the goth/gothic session during the process of linking the external user.

Reply:
d6ad03349a3a6f11ff67e4e0f5c0bfbce4eb2708